blocks/edit/da-title/da-title.js

```diff
@@ -33,6 +33,32 @@ export default class DaTitle extends LitElement {
     inlinesvg({ parent: this.shadowRoot, paths: ICONS });
   }
 
+  async saveVersion(pathname) {
+    const edit = this.parentNode.querySelector('da-content').shadowRoot.querySelector('da-editor');
+    if (!edit) {
+      // eslint-disable-next-line no-console
+      console.log('Not able to obtain document path');
+      return;
+    }
+    // TODO there must be a better way to obtain path
+    const url = new URL(edit._path);
```

Comment:
@auniverseaway do you know a cleaner way to obtain the `sourceURL` to the currently edited do than this?